index.py

```diff
@@ -53,18 +55,21 @@ def build(self):
         Punctuation handling, tokenisation, case-folding, stemming are applied to generate terms
         """
         # Step 1: Obtain Postings for material to create VSM in Step 3
-        tokens_list = []
         set_of_documents = self.get_documents()
+        tokens_list = []
+
         # Save flattened Counter results in tokens_list
         for res in set_of_documents:
             doc_id = res['doc_id']
-            positional_indexes = res['positional_indexes']
-            for term, positions in positional_indexes.items():
-                tokens_list.append([term, (doc_id, positions)])
+            # Generate tokens of positional indexes (subsequently form into posting list)
```

Comment:
Nice idea, could we also add an additional line in the comparator function in line 27 to sort by `Field` enum also? So that when we do the index generation, there's an order to the postings in the posting list

Reply:
ok! in this case which field should come first? or would it be ok as long as there's an order to it?

Reply:
I think any order works haha, just need one